asm/f64/add_amd64.s

```diff
@@ -4,13 +4,13 @@
 
 #include "textflag.h"
 
-// func Daddconst(dst, s []float64)
+// func Add(dst, s []float64)
 TEXT ·Add(SB), NOSPLIT, $0
 	MOVQ 	dst_base+0(FP), DI
 	MOVQ 	dst_len+8(FP), DX
 	MOVQ 	s_base+24(FP), SI
 	CMPQ	s_len+32(FP), DX
-	CMOVLEQ	s_len+32(FP), DX
+	CMOVQLE	s_len+32(FP), DX
```

Comment:
It was "working" for all tests with `s` as the shortest slice.  Instead of moving an int64 if less than or equal, it was moving an int32 if equal.  Not a fun bug, as it's as easy as putting the size `Q` before or after the condition `LE`.  Started as a typo that got copied elsewhere.